src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -32,7 +32,7 @@
          to either invoke methods on the daemon, or monitor status.
          If no clients are registered, the daemon may exit.
 
-         No options are currently defined.
+         'name' (type 's') - Package/component name (e.g. `cockpit`, `gnome-software`)
```

Comment:
`s/name/id/`?

Reply:
Yeah, I noticed that in self-review, had already pushed a fixup
.

Reply:
Ahh right, sigh. I tend to do reviews by looking at each commit one at a time, and I missed the fixup! WDYT if in general we just squash fixups into the commits as long as the PR hasn't been reviewed? Fixups are really helpful for follow-up reviews, but demand extra processing for initial reviews.